### PR TITLE
Update strings and add developer preferences for Suggested edits

### PR DESCRIPTION
**Pull request by cooltey**
`#6` and `#7` in https://phabricator.wikimedia.org/T225635

Also added developer preferences for showing image captions dialogs and notification for testing purposes.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -690,30 +690,30 @@
...
-    <string name="suggested_edits_unlock_translate_captions_notification_big_text">Suggested edits\n\nThanks for all your contributions. You’re using the app in multiple languages, so we’ve compiled a list of images needing caption translations. Will you continue to help us improve Wikipedia?</string>
+    <string name="suggested_edits_unlock_translate_captions_dialog_message">Congratulations on your 15th image caption edit, we’re amazed! You’re using the app in multiple languages, so we’ve compiled a list of images needing caption translations. Could you continue to help us improving Wikipedia?</string>
+    <string name="suggested_edits_unlock_translate_captions_notification_big_text">Suggested edits\n\nCongratulations on your 15th image caption edit, we’re amazed! You’re using the app in multiple languages, so we’ve compiled a list of images needing caption translations. Could you continue to help us improving Wikipedia?</string>
```
Comment (dbrant) [inline app/src/main/res/values/strings.xml]: Let's not put any absolute numbers ("15th") into these strings. There is a 100% chance that these numbers will change later, and they should not be hard-coded into the translations.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -690,30 +690,30 @@
...
-    <string name="suggested_edits_unlock_add_captions_dialog_message">Thanks for all your contributions! Since we think you like adding captions to images, we’ve compiled a list of images that are missing captions. Could you help us improve Wikipedia?</string>
-    <string name="suggested_edits_unlock_add_captions_notification_big_text">Suggested edits\n\nThanks for all your contributions. Since we think you like adding captions to images, we’ve compiled a list of images that are missing captions. Could you help us improve Wikipedia?</string>
+    <string name="suggested_edits_unlock_add_captions_dialog_message">Congratulations on your third image caption edit! Since we think you like adding captions to images, we’ve compiled a list of images missing captions. Could you help us improving Wikipedia?</string>
+    <string name="suggested_edits_unlock_add_captions_notification_big_text">Suggested edits\n\nCongratulations on your third image caption edit! Since we think you like adding captions to images, we’ve compiled a list of images missing captions. Could you help us improving Wikipedia?</string>
```
Comment (dbrant) [inline app/src/main/res/values/strings.xml]: Let's not put any absolute numbers ("third") into these strings. There is a 100% chance that these numbers will change later, and they should not be hard-coded into the translations.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (cooltey) [reply app/src/main/res/values/strings.xml]: Make sense. Will update this after getting new strings.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (cooltey) [reply app/src/main/res/values/strings.xml]: Done

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (cooltey) [reply app/src/main/res/values/strings.xml]: Done.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -690,30 +690,30 @@
     <string name="suggested_edits_unlock_notification_button">Yes, let’s go</string>
     <string name="suggested_edits_unlock_add_captions_dialog_title">You unlocked the "Add image captions" task</string>
     <string name="suggested_edits_unlock_add_captions_notification_title">You unlocked "Add image captions"</string>
-    <string name="suggested_edits_unlock_add_captions_dialog_message">Thanks for all your contributions! Since we think you like adding captions to images, we’ve compiled a list of images that are missing captions. Could you help us improve Wikipedia?</string>
-    <string name="suggested_edits_unlock_add_captions_notification_big_text">Suggested edits\n\nThanks for all your contributions. Since we think you like adding captions to images, we’ve compiled a list of images that are missing captions. Could you help us improve Wikipedia?</string>
+    <string name="suggested_edits_unlock_add_captions_dialog_message">Thanks for all your image caption edits! Since we think you like adding captions to images, we’ve compiled a list of images missing captions. Could you help us improving Wikipedia?</string>
```
Comment (dbrant) [inline app/src/main/res/values/strings.xml]: `Could you help us improving Wikipedia?` is incorrect grammar.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -690,30 +690,30 @@
...
+    <string name="suggested_edits_unlock_add_captions_notification_big_text">Suggested edits\n\nThanks for all your image caption edits! Since we think you like adding captions to images, we’ve compiled a list of images missing captions. Could you help us improving Wikipedia?</string>
```
Comment (dbrant) [inline app/src/main/res/values/strings.xml]: `Could you help us improving Wikipedia?` is incorrect grammar.

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -690,30 +690,30 @@
...
     <string name="suggested_edits_unlock_translate_captions_dialog_title">You unlocked the "Translate image captions" task</string>
     <string name="suggested_edits_unlock_translate_captions_notification_title">You unlocked "Translate image captions"</string>
-    <string name="suggested_edits_unlock_translate_captions_dialog_message">Thanks for all your contributions! You’re using the app in multiple languages, so we’ve compiled a list of images needing caption translations. Will you continue to help us improve Wikipedia?</string>
-    <string name="suggested_edits_unlock_translate_captions_notification_big_text">Suggested edits\n\nThanks for all your contributions. You’re using the app in multiple languages, so we’ve compiled a list of images needing caption translations. Will you continue to help us improve Wikipedia?</string>
+    <string name="suggested_edits_unlock_translate_captions_dialog_message">Congratulations on your many image caption edits, we’re amazed! You’re using the app in multiple languages, so we’ve compiled a list of images needing caption translations. Could you continue to help us improving Wikipedia?</string>
```
Comment (dbrant) [inline app/src/main/res/values/strings.xml]: `Could you continue to help us improving Wikipedia?` is incorrect grammar.

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -690,30 +690,30 @@
...
+    <string name="suggested_edits_unlock_translate_captions_notification_big_text">Suggested edits\n\nCongratulations on your many image caption edits, we’re amazed! You’re using the app in multiple languages, so we’ve compiled a list of images needing caption translations. Could you continue to help us improving Wikipedia?</string>
```
Comment (dbrant) [inline app/src/main/res/values/strings.xml]: Same.

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (cooltey) [reply app/src/main/res/values/strings.xml]: Done.
https://phabricator.wikimedia.org/T225635#5288605

**Comment by cooltey** [comment]
Updated the strings:
```
You unlocked the “Add image captions” task
Thanks for all your image caption edits! Since we see you like adding captions to images, we’ve compiled a list of images missing captions. Could you help us improve Wikipedia?

You unlocked the “Translate image captions” task
Congratulations on your many image caption edits - we’re amazed! You’re using the app in multiple languages, so we’ve compiled a list of images needing caption translations. Could you continue to help us improve Wikipedia?
```
https://phabricator.wikimedia.org/T225635#5288605

**Pull request merged**